frontend/src/screens/Wishlist/index.js

```diff
@@ -1,26 +1,43 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import useCustomContext from '../../customHooks/Hook';
-import { REMOVE_FROM_WISHLIST } from '../../constants/WishListConstants';
-import { ADD_TO_CART } from '../../constants/CartConstants'
 import { Message, Rating } from '../../components';
 import { Link } from 'react-router-dom';
 import styles from './wishlist.module.css';
+import axios from 'axios';
+import { auth } from '../../utlis/auth';
+import Config from '../../config/Config';
+import { REMOVE_FROM_WISHLIST, ADD_TO_CART } from '../../constants/type';
+import { toast } from 'react-toastify';
+
+const { serverUrl } = Config
 
 const WishList = () => {
+    const token = auth()
     const { state: { wishlist, cart }, dispatch } = useCustomContext();
     const [qty, setQty] = useState(1);
 
-    const removefromWishList = (id) => {
-        dispatch({ type: REMOVE_FROM_WISHLIST, payload: id })
+    const removefromWishList = async (id) => {
+        try {
+            const { data } = await axios.delete(`${serverUrl}/api/wishlists/${id}`, { headers: token })
+            dispatch({ type: REMOVE_FROM_WISHLIST, payload: data.productsinWishlist })
+        } catch (err) {
+            const error = err.response.data.message;
+            toast.error(`${error}`);
+        }
     }
 
-    const addToCart = (item) => {
-        const updateQtyProduct = { ...item, qty: qty }
-        dispatch({ type: ADD_TO_CART, payload: updateQtyProduct })
+    const addToCart = async (item) => {
+        try {
+            const { data } = await axios.post(`${serverUrl}/api/carts`, { 'productId': item._id, 'quantity': qty }, { headers: token });
+            dispatch({ type: ADD_TO_CART, payload: data.productsinCart })
+        } catch (err) {
+            const error = err.response.data.message;
+            toast.error(`${error}`);
+        }
     }
```

Comment:
AddToCart has been used in multiple screens. You could create a separate file for this and reuse it across screens.

Reply:
That's really a great idea.
done